MeshLib/MeshEditing/MeshRevision.cpp

```diff
@@ -67,39 +67,68 @@ MeshLib::Mesh* MeshRevision::simplifyMesh(const std::string &new_mesh_name, doub
 	if (this->_mesh.getNElements() == 0)
 		return nullptr;
 
+	// data structures for the new mesh
 	std::vector<MeshLib::Node*> new_nodes = this->constructNewNodesArray(this->collapseNodeIndices(eps));
 	std::vector<MeshLib::Element*> new_elements;
-
-	const std::vector<MeshLib::Element*> &elements(this->_mesh.getElements());
-	for (auto elem = elements.begin(); elem != elements.end(); ++elem)
-	{
-		unsigned n_unique_nodes(this->getNUniqueNodes(*elem));
-		if (n_unique_nodes == (*elem)->getNBaseNodes() && (*elem)->getDimension() >= min_elem_dim)
+	MeshLib::Properties new_properties;
+	boost::optional<PropertyVector<int> &> new_material_vec(
+		new_properties.createNewPropertyVector<int>(
+			"MaterialIDs", MeshItemType::Cell, 1
+		)
```

Comment:
just a formatting thing:
if you write
```c++
boost::optional<PropertyVector<int> &> new_material_vec(
		new_properties.createNewPropertyVector<int>("MaterialIDs", MeshItemType::Cell, 1)
```
it is still less than 100 chars per line (which is our limit) but uses only 2 lines for the command instead of 4.
I don't know if it's just me, but i always get very confused if i see things that should/could be on one line subdivided into multiple lines ... and the more lines the more confusing ....
if you don't mind too much it would be really helpful if you could use the 100 char limit, there are multiple instances in this class alone where you would save a bunch of lines and (in my opinion) would make the source more easily comprehendable.